### PR TITLE
Bump version to 0.11.0

### PR DESCRIPTION
**Pull request by ekoval**
#59

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # [Changelog](https://github.com/yola/sitewit/releases)
 
-## DEV
+## 0.11.0
```
Comment (RayeN) [inline CHANGELOG.md]: We could probably bump only patch version here?

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**